src/lib/content/sodium/modules/AbemaTVLiveTypeHandler.js

```diff
@@ -227,7 +227,7 @@ export default class AbemaTVLiveTypeHandler extends GeneralTypeHandler {
 
   get_video_title() {
     try {
-      return document.querySelector('.com-tv-SlotHeading__title').textContent;
+      return document.querySelector('.com-m-TextIcon--dark:not([class*=" "])').textContent;
```

Comment:
少し整理しておきたいので確認させてください。
この変更したコードの意図としては以下のいずれでしょうか:

- `class="com-m-TextIcon--dark"` 属性の要素のみ
- `class="com-m-TextIcon--dark …ほかにも1つ以上複数存在する"` 属性の要素のみ
- 上記いずれかの要素

Reply:
> class="com-m-TextIcon--dark" 属性の要素のみ

意図として上記です
理由としては、com-m-TextIcon--dark要素が2つあり、
`class="com-m-TextIcon--dark"` 属性の要素のみの方がビデオタイトルとして適切だと思ったからです。

Reply:
なるほどです。その場合、次のように属性一致 `=` で書くとより明確かと感じました。ご参考までに:

```suggestion
      return document.querySelector('[class="com-m-TextIcon--dark"]').textContent;
```

Reply:
ありがとうございます。
参考にさせていただきます

Reply:
https://github.com/videomark/videomark/pull/246/commits/f4c82dad7dd5490ca49a79d3eac2f419b72bf88d で修正しました